src/controllers/HighlightModeController.js

```diff
@@ -47,6 +54,28 @@ class HighlightModeController extends AnnotationModeController {
         this.emit(CONTROLLER_EVENT.unbindDOMListeners); // Disable other annotations
         this.bindListeners(); // Enable mode
     }
+
+    /**
+     * Renders annotations from memory for a specified page.
+     *
+     * @inheritdoc
+     * @private
+     * @param {number} pageNum - Page number
+     * @return {void}
+     */
+    renderPage(pageNum) {
+        // Clear context if needed
+        const pageEl = this.annotatedElement.querySelector(`[data-page-number="${pageNum}"]`);
+        const layerClass =
+            this.mode === TYPES.highlight ? CLASS_ANNOTATION_LAYER_HIGHLIGHT : CLASS_ANNOTATION_LAYER_HIGHLIGHT_COMMENT;
```

Comment:
so if highlights are enabled then target the plain canvas otherwise target the highlight comment canvas? What happens if we create a plain highlight and then add a comment to it. Does it get moved to a different canvas?

Reply:
Separating this change from the bug fix since having 2 different canvases doesn't affect the bug caused in #71